docker/base/setup_clusterfuzz.sh

```diff
@@ -37,14 +37,14 @@ if [[ -z "$DISABLE_MOUNTS" ]]; then
   mount -t tmpfs -o size=10M,mode=777 tmpfs $INSTALL_DIRECTORY/clusterfuzz/bot/logs/
   mount -t tmpfs -o size=90M,mode=777 tmpfs $BOT_TMPDIR
 
-  # Setup mount to limit disk space for fuzzer-testcases-disk directory.
+  # Setup mount to limit disk space for fuzzer testcases disk directory.
   FUZZER_TESTCASES_DISK_FILE=$INSTALL_DIRECTORY/fuzzer-testcases.mnt
   fallocate -l 8GiB $FUZZER_TESTCASES_DISK_FILE
   mkfs.ext4 -F $FUZZER_TESTCASES_DISK_FILE
 
   # mkfs.ext4 seems to remove the previous allocation, so do it again.
   fallocate -l 8GiB $FUZZER_TESTCASES_DISK_FILE
-  mount -o loop $FUZZER_TESTCASES_DISK_FILE $INSTALL_DIRECTORY/clusterfuzz/bot/inputs/fuzzer-testcases-disk
+  mount -o loop $FUZZER_TESTCASES_DISK_FILE $INSTALL_DIRECTORY/clusterfuzz/bot/inputs/disk
```

Comment:
Yes - I think I saw a comment from you elsewhere (maybe a chat room?) saying that it might be better to flatten the whole structure a bit. Would you prefer that?

Also I'm happy with any other solution that feeds a shorter `TMPDIR` to Chromium. If there's some other way, e.g. giving it a `TMPDIR` inside `/tmp` instead of in a deeply nested path, that's fine with me - let me know what you'd like me to do.

Reply:
Let's just call it ft-disk for now.